backend/src/backend/routes/titles.py

```diff
@@ -40,4 +40,5 @@ async def get_title(ident: str):
             )
             for item in await title.metadata.all()
         },
+        "books": [book.id for book in title.books],
```

Comment:
I suggest we return `[{"id": book.id, "name": book.metadata.get(name="Name")}` instead.
This will be useful in any UI-driven context as the Id is not human friendly and would otherwise require requesting the full book endpoint just to get a hint of what's behind the Id.

Reply:
@rgaudin I have this snippet, let me know if this is okay by you
```py
    books = []
    if title.books:
        for book in title.books:
            name = await book.metadata.get(name="Name")
            books.append({"id": book.id, "name": name.value})
```
because `book.metadata.get(name="Name")` returns a queryObject of type `BookMetadata` and I am getting a `ValidationError` from pyDantic when I do

```
await book.metadata.fields("fields").get(name="Name")
```

Reply:
Yes, mine was pseudo-code. You should add a `name` property on the book instance that returns the metadata value directly (or None if not present). That seems adequate/useful.

Reply:
hi @rgaudin I could not get a one-line solution to work for this, let me know if the current solution works, I have made the commit for you to review